### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by 1GORDON**
# Hello Microverse project
In this project, I added the following files
* index.html file
* style.css file
* gitignore file
* README file
* test.md file

I added a test,dm file to the .gitignore and updated my linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# gitflow
\ No newline at end of file
+# gitflow
```
Comment (DanteAlonsoHT) [inline README.md:4]: One of the most important parts of any project is the README file, be sure your README file is descriptive and complete according to your project, take into consideration that not always you should use all of the sections provided by the [Template of Microverse](https://github.com/microverseinc/readme-template), it depends of the projects.

- [ ] Use the [Template of Microverse for README files](https://github.com/microverseinc/readme-template) and fill it with your information about you and your project.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @1GORDON, 

You did an amazing job, your workflow, your code is good!
There is an important issue that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

<hr>

I know you have good skills to complete these changes required in time 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (1GORDON) [reply README.md:4]: Thank you. I am still trying to get used to the new system. Thank you again. Let me work on it
I hope it doesn't affect the number of reviews I am left with

Comment (1GORDON) [reply README.md:4]: I managed to add one extra line in the README.md file. Is it fine or I add the other sections?

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
-# gitflow
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# gitflow
```
Comment (mke2111) [inline README.md:4]: - [x] [OPTIONAL] Add an appropriate project title. This will give an overall view of the project.

**Review by mke2111 — APPROVED** [review]
### Hi @1GORDON, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 1GORDON** [comment]
Thank you for the positive news. I was so desperate for a positive result. Thank again

**Pull request merged**